packages/hash/external-services/kratos/kratos.yml

```diff
@@ -0,0 +1,61 @@
+# Based on the quick-start example https://github.com/ory/kratos/blob/master/contrib/quickstart/kratos/email-password/kratos.yml
+
+version: v0.7.1-alpha.1
```

Comment:
Kratos logs spews out this message:
"`Config version is 'v0.7.1-alpha.1' but kratos runs on version 'v0.10.1'`"
Can we use the newer version that mates the version of kratos we're using or is there a reason for this version?